docs/getting-started/configuring-your-repository.md

```diff
@@ -0,0 +1,41 @@
+# Configuring your repository
+
+Once you've [added your first repository](codacy-quickstart.md), it's important that you configure the analysis tools to match the use cases of your team, enforce or promote any coding conventions and best practices that your team may already be following, and avoid time-consuming false positives.
```

Comment:
Here are some suggestions to round things off:

- The "use cases of the team" (unless we have some other ideas), are actually the existing conventions or best practices of the team, and any new ones that the user configuring the repository may want to introduce. So we can rephrase to include these as examples instead.
- I like your idea of "promoting" best practices/conventions, and I think we can replace "enforce" with that since it has a less negative connotation.
- Finally, I suggest that we make it more explicit why the false positives can be distracting and time-consuming. This is the kind of argumentation that should convince the users not to skip performing these configurations.

```suggestion
Once you've [added your first repository](codacy-quickstart.md), it's important that you configure the analysis tools to match the use cases of your team, such as configuring any coding conventions and best practices that your team may already be following or that you want to promote. It's also critical to review the configurations to avoid reporting false positives or any other issues that don't bring value to your team, which can introduce unwanted delays to the development process.
```